packages/launchpad/cypress/e2e/migration.cy.ts

```diff
@@ -81,23 +150,31 @@ describe('Migration', { viewportWidth: 1200 }, () => {
         }
       })
     })
-  })
 
-  describe('Full flow', () => {
-    it('goes to each step', () => {
+    it('renames support file', () => {
+      cy.visitLaunchpad()
+      cy.findByText(`Rename these specs for me`).click()
+      cy.findByText(`I'll do this later`).click()
+      cy.findByText(`Rename the support file for me`).click()
+
       cy.withCtx(async (ctx) => {
-        await ctx.actions.file.writeFileInProject('cypress.json', '{}')
+        expect(
+          await ctx.actions.file.checkIfFileExists(ctx.path.join('cypress', 'support', 'e2e.js')),
+        ).not.to.be.null
       })
+    })
+  })
 
+  describe('Full flow', () => {
+    it('goes to each step', () => {
       cy.visitLaunchpad()
 
       cy.findByText(defaultMessages.migration.wizard.step1.button).click()
       cy.findByText(defaultMessages.migration.wizard.step2.button).click()
       cy.findByText(defaultMessages.migration.wizard.step3.button).click()
       cy.findByText(defaultMessages.migration.wizard.step4.button).click()
       cy.findByText(defaultMessages.migration.wizard.step5.button).click()
-
-      // cy.findByText(defaultMessages.setupWizard.selectFramework.description).should('be.visible')
+      cy.findByText('Welcome to Cypress!').should('be.visible')
```

Comment:
Do we want to continue to click through the launchpad and launch the app to verify that it works all the way?

Reply:
Probably a good idea to add more long-form test cases.